.circleci/config.yml

```diff
@@ -136,3 +144,17 @@ jobs:
           name: Build - Notify failure
           when: on_fail
           command: notify-job-failure.sh
+
+  rips-test:
+    docker:
+      - image: gcr.io/planet-4-151612/p4-builder:latest
+    working_directory:  /home/circleci/
+    environment:
+      RIPS_SCAN_REPO: planet4-master-theme
+      RIPS_SCAN_APP_ID: 32
+    steps:
+      - setup_remote_docker:
```

Comment:
This job doesn't require a docker machine, I believe. Removing this line will save up to a minute of waiting for a docker machine to be provisioned.